src/explore-education-statistics-admin/src/pages/publication/components/PublicationForm.tsx

```diff
@@ -139,7 +143,7 @@ const PublicationForm = ({
               />
             )}
 
-            <FormFieldTextInput<FormValues>
+            <FormFieldTextArea<FormValues>
```

Comment:
I think you could use `maxLength={160}` here

Reply:
Yep, good call